### ISSUE TITLE
Create a process for bringing project by lesser-known Scratchers to the surface

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#4005]
Many Scratchers in the community have complained over the years that it is hard for most Scratchers to get even a small amount of attention to their work. 

It has been observed that there is a very small "middle class" in terms of the Scratch community's attention -- either a project gets featured and gets tons of attention, or it exists in obscurity and is lucky to get a single favorite/like.

This project, for example, expresses some of the frustration (though the particular proposal may not be the right solution): https://scratch.mit.edu/projects/396713463

This is a large, long-term issue, that would involve both technical development, design, and community team processes.

Some possible ideas:
* Create an "up and coming" project list visible to users
    * there are various ways this could be generated, either manually or automatically
    * This could include:
        * indications that the user is legit and does sustained work (length of time registered, number of projects created, history of being reported, number of others' projects user has commented on and liked, complexity of original projects, etc.)
        * indications that the project is the result of sustained work (complexity of project, presence of original assets, etc.). 
* Expand "front page curator" role to allow approved users to flag projects and users as making high-quality, safe, non-meme/troll work

**Comment by R4356th** [comment]
@benjiwheeler, accepting user ideas?

**Comment by Kenny2github** [comment]
For a long time there was a "Recent projects" row on the homepage that was at least somewhat successful in bringing attention to projects that would otherwise never be seen; however, that was [removed][1] because it wasn't doing that well, and more importantly had the potential to show inappropriate content on the homepage.
Whatever solution you end up going with must be if not 100% at least like 99.9% effective at not showing inappropriate projects, or it may suffer the same fate.

[1]: https://github.com/LLK/scratch-www/issues/1860

**Comment by benjiwheeler** [comment]
> Whatever solution you end up going with must be if not 100% at least like 99.9% effective at not showing inappropriate projects, or it may suffer the same fate.

I agree, somewhat. This is tricky, for sure -- we would not want to give a large amount of attention to a relatively new user, with relatively few projects, whose work hasn't been vetted somehow.

That said, it might be possible to use a system that gives brief attention to a wide variety of projects, which are not random but reflect various indications of likely trustworthiness.

**Comment by Accio1** [comment]
I feel like a list of somewhat flexible conditions would be good for preventing inappropriate projects from being displayed.

For example:

- The user isn't a New Scratcher/their account is at least 1 month old
- The user has shared at least three projects
- The user hasn't had any alerts within the past month
- The project is at least 3 days old and has at least 5 views
- The project has a certain amount of assets (scripts, sprites, costumes, sounds, etc.)

I also like the idea of having a program similar to the FPC program. Perhaps there could be a form, and users could propose 1 project of theirs every 2 weeks, which could then be reviewed by a team of Scratchers and marked as either approved or denied.

Another idea that I thought of that would combine both of the above ideas is that Scratchers could "report" projects/users to be shown to the community. If the project fit the criteria in the list above, it would be sent to be approved or denied by the team of selected users.

This feature would be fantastic if it was added, because not only would it bring lesser known Scratchers to the attention of the community, but it would probably also cut down on the amount of advertising happening in inappropriate places (project pages, profile pages, the wrong forums, etc.).

**Comment by Kenny2github** [comment]
> Scratchers could "report" projects/users to be shown to the community.

This makes so much sense! Sort of like reporting a forum topic to be stickied. :) I like this idea very much - sort of a nerfed "featured projects" section.

**Comment by apple502j** [comment]
This is my easy-to-implement idea: Don't show 1st and 2nd page of Explore. It will still require some views and loves to get onto Explore page, and if the project becomes too popular it will disappear.

**Comment by benjiwheeler** [comment]
@apple502j that's smart! A related idea would be to introduce some randomness into the explore page, so projects that only have a handful of likes sometimes show up at the top.

**Comment by JeffaloBob** [comment]
> @apple502j that's smart! A related idea would be to introduce some randomness into the explore page, so projects that only have a handful of likes sometimes show up at the top.

but this would cause more people to complain about how their projects dont make it to the explore page. this is something to consider

**Comment by R4356th** [comment]
> Perhaps there could be a form, and users could propose 1 project of theirs every 2 weeks, which could then be reviewed by a team of Scratchers and marked as either approved or denied.

I proposed that idea in the discussion forums and it got rejected for two reasons- Scratch is supposed to be a helpful community and doing this will not let the reality be so and the current way prevents inappropriate contents to get on the front page.


> @apple502j that's smart! A related idea would be to introduce some randomness into the explore page, so projects that only have a handful of likes sometimes show up at the top.

Some users like inappropriate contents a lot, too. So that is not a good idea considering the consequences.

**Comment by Accio1** [comment]
> > Perhaps there could be a form, and users could propose 1 project of theirs every 2 weeks, which could then be reviewed by a team of Scratchers and marked as either approved or denied.
> 
> I proposed that idea in the discussion forums and it got rejected for two reasons- Scratch is supposed to be a helpful community and doing this will not let the reality be so and the current way prevents inappropriate contents to get on the front page.

@R4356th I'm a bit confused by your reply to this. Could you please elaborate or link to the post in the discussion forums so I can see the exact wording of why it was rejected?

**Comment by R4356th** [comment]
@Accio1, here you go- https://scratch.mit.edu/discuss/topic/400079/?page=4#post-3983330 .

**Comment by Accio1** [comment]
> @Accio1, here you go- https://scratch.mit.edu/discuss/topic/400079/?page=4#post-3983330 .

The thing that suggested above is different from that suggestion. I was proposing a completely separate section on the homepage (similar to the recently shared projects section from Scratch 2.0). Users would propose their projects to go there, and the featured project row would remain the same as it is now.

**Comment by R4356th** [comment]
> > @Accio1, here you go- https://scratch.mit.edu/discuss/topic/400079/?page=4#post-3983330 .
> 
> The thing that suggested above is different from that suggestion. I was proposing a completely separate section on the homepage (similar to the recently shared projects section from Scratch 2.0). Users would propose their projects to go there, and the featured project row would remain the same as it is now.

Well, I did understand your suggestion but I do not see how it would be something different and how the same reasons why my suggestion got rejected do not apply there.

**Comment by CubeyTheCube** [comment]
Not sure how well known this is... you can go to https://scratch.mit.edu/explore and click the button that says "Trending" and set it to "Recent". It shows recently created projects, regardless of how many views they get.
